packages/react-instantsearch-core-next/rollup.config.js

```diff
@@ -0,0 +1,74 @@
+import babel from 'rollup-plugin-babel';
+import commonjs from 'rollup-plugin-commonjs';
+import filesize from 'rollup-plugin-filesize';
+import globals from 'rollup-plugin-node-globals';
+import resolve from 'rollup-plugin-node-resolve';
+import replace from 'rollup-plugin-replace';
+import { uglify } from 'rollup-plugin-uglify';
+
+const clear = x => x.filter(Boolean);
+
+const version = process.env.VERSION || 'UNRELEASED';
+const algolia = '© Algolia, inc.';
+const link = 'https://github.com/algolia/react-instantsearch';
+const createLicence = name =>
```

Comment:
That's not a license (MIT in our case) but a banner.

```suggestion
const createBanner = name =>
```